pkg/host/internal/bridge/ovs/ovs.go

```diff
@@ -148,15 +148,36 @@ func (o *ovs) CreateOVSBridge(ctx context.Context, conf *sriovnetworkv1.OVSConfi
 		funcLog.Error(err, "CreateOVSBridge(): failed to get bridge after creation")
 		return err
 	}
-	funcLog.V(2).Info("CreateOVSBridge(): add internal interface to the bridge")
-	if err := o.addInterface(ctx, dbClient, bridge, &InterfaceEntry{
-		Name: bridge.Name,
-		UUID: uuid.NewString(),
-		Type: "internal",
-	}); err != nil {
-		funcLog.Error(err, "CreateOVSBridge(): failed to add internal interface to the bridge")
+	funcLog.V(2).Info("CreateOVSBridge(): Check if internal interface exists in the bridge")
```

Comment:
this function is getting pretty long, WDYT to split out internalinterface creation to a separate func ?

Reply:
e.g: `o.addInternalInterface(...) error`

Reply:
Done